docs/source/notes/cuda.rst

```diff
@@ -206,6 +208,24 @@ To toggle the reduced precision reduction flags in C++, one can do
 
   at::globalContext().setAllowBF16ReductionCuBLAS(true);
 
+.. _fp16accmulation:
+
+Full FP16 Accmumulation in FP16 GEMMs
+-------------------------------------
+
+Certain GPUs have increased performance when doing _all_ FP16 GEMM accumulation
+in FP16, at the cost of numerical precision. This behavior can be enabled via:
+
+.. code:: python
+
+  torch.backends.cuda.matmul.use_fp16_accumulation = True
```

Comment:
I think it would be good to align these two flags a bit more. Perhaps just `allow_fp16_reduced_precision_accumulation`?

Reply:
yeah I think that makes sense, was initially waffling about this as technically the reduction flag is leaving it up to heuristics whether or not reduced precision reductions were used, but fp16 accumulation is more heavy-handed in that it requires scales to be in fp16 as well (I was thinking that implies that it's _always_ going to use fp16 accum)